root/static/css/style.css

```diff
@@ -1396,6 +1396,14 @@ a.curs-editable-table-field img, a.curs-editable-table-long-field img
   display: inline;
 }
 
+.curs-annotation-edit .curs-edit-wide-field {
+  width: 100%;
```

Comment:
So I guess the plan is to only enable scaling on the wide fields? That's fine by me, since they're the ones that are most likely to cause overflow problems.